internal/packages/archetype/input_agent_config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package archetype
+
+const inputAgentConfigTemplate = `data_stream:
+  dataset: ` + "{{`{{data_stream.dataset}}`}}" + `
```

Comment:
Cannot we avoid these concatenations, they make these templates difficult to read.

Reply:
It could be changed to:

```suggestion
  dataset: {{ "{{data_stream.dataset}}" }}
```

It looks at least somehow better to read, no ? @jsoriano .

I could update the other templates to use this way of writing those strings.

Reply:
Just updated all the templates to use that syntax to write strings with `{{}}`

Moreover, all these templates and resources (logo and screenshot) have been moved to a `_static` folder and they are loaded now using `embed`